mlrun/feature_store/api.py

```diff
@@ -410,6 +410,18 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "feature set and source must be specified"
         )
+    if not targets:
+        if not isinstance(featureset, FeatureSet):
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "targets must be specified\n"
+                "If you don't want to provide any target use `preview` function instead"
+            )
+        elif not (featureset.spec.targets or featureset.spec.with_default_targets):
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                f"You disabled the default targets of the {featureset.metadata.name} feature set "
```

Comment:
please repharse without using "you". Something like:
no targets provided to feature set {featureset.metadata.name} ingest, aborting.\n
(`preview` can be used as an alternative to local ingest when targets are not needed)

"If you don't want to provide any target use preview function instead"